engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -25,9 +25,39 @@ object WorkflowExecutionActor {
   case object WorkflowExecutionAbortedState extends WorkflowExecutionActorTerminalState
 
   /**
-    * State data
+    * State Data
+    * @param callQueue The ordered (and grouped) List of Lists with safe ordering among them
+    * @param callOutputs A Map of Call -> Outputs
+    * @param inFlight Set of calls that are currently running
     */
-  final case class WorkflowExecutionActorData()
+  final case class WorkflowExecutionActorData(private var callQueue: List[List[Scope]],
+                                              private var callOutputs: Map[Call, cromwell.core.CallOutputs],
```

Comment:
I totally agree. And what used to be the executionStore (or the execution graph, or whatever) should be in the stateData too.

Hooray for reducing the amount of hidden mutable state :D